atomic_reactor/plugins/pre_change_from_in_df.py

```diff
@@ -80,35 +76,17 @@ def run(self):
         if missing_set:
             # this would indicate another plugin modified parent_images out of sync
             # with the Dockerfile or some other code bug
-            raise ParentImageMissing("Lost parent image(s) from Dockerfile: {}".format(missing_set))
+            raise ParentImageMissing("Lost parent image(s) from Dockerfile images: {}".format(
+                missing_set))
```

Comment:
Would it be more readable by using f-string here?

Reply:
Updated.